### PR TITLE
Story6 countdown

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: css/game.css [hunk 1]
```diff
@@ -86,6 +86,13 @@ header {
     margin-bottom: 5px;
 }
 
+.start-countdown {
+    font-size: 10rem;
+    color: #ff0000;
+    text-shadow: 1px 4px 5px black;
```
Comment (mporam) [inline css/game.css]: hex colors for consistency please

File: js/game.js [hunk 2]
```diff
@@ -93,7 +95,13 @@ function fadeTables(tables) {
  */
 function showFinalScore() {
     gameOver.classList.remove('hidden')
-    gameOver.innerHTML = '<p>Game Over!</p><p class="final-score">You whacked ' + score + ' rats</p>'
+    if (score === 1) {
+        gameOver.innerHTML = '<p>Game Over!</p><p class="final-score">You whacked ' + score + ' rat</p>'
```
Comment (mporam) [inline js/game.js]: inside your if have a variable called `msg` which changes in each statement, then set the html after the ifs using:
```
gameOver.innerHTML = '<p>Game Over!</p><p class="final-score">' + msg + '</p>'
```

File: js/game.js [hunk 3]
```diff
@@ -128,9 +155,11 @@ function countDown() {
 function startClock() {
     countDownId = setInterval(countDown, 1000)
 }
+function startCountDownInterval() {
```
Comment (mporam) [inline js/game.js:166]: missing line break and docblock

**Review by mporam — CHANGES_REQUESTED** [review]

File: css/game.css (hunk 1 above)
Comment (alecorra) [reply css/game.css]: done

File: js/game.js (hunk 2 above)
Comment (alecorra) [reply js/game.js]: fixed, code dry

File: js/game.js (hunk 3 above)
Comment (alecorra) [reply js/game.js:166]: fixed

**Review by mporam — APPROVED** [review]

**Pull request merged**